src/Features/Core/Portable/SolutionCrawler/GlobalOperationAwareIdleProcessor.cs

```diff
@@ -15,59 +15,28 @@ internal abstract class GlobalOperationAwareIdleProcessor : IdleProcessor
     {
         private readonly IGlobalOperationNotificationService _globalOperationNotificationService;
 
-        private TaskCompletionSource<object?>? _globalOperation;
-        private Task _globalOperationTask;
```

Comment:
instead of a separate tracking trask, the logic for pausing moved up into the base class (IdleProcessor).